src/decorators/is-date.spec.ts

```diff
@@ -29,12 +34,39 @@ describe('IsDate', () => {
         Result.err('date is not formatted as `yyyy-mm-dd`')
       );
     });
+
+    it('transforms to date formatted string', async () => {
+      const dateToUse = new Date();
+      const obj = new TestSerialize();
+      obj.date = dateToUse;
+
+      expect(await output(obj)).toStrictEqual({ date: dateToUse.toISOString().slice(0, 10) });
+    });
+
+    it('pass through when field is undefined', async () => {
```

Comment:
I think this should throw, there is no 'optional' property in the parameters

Reply:
Yep, I agree that i should throw, not silently pass through the value